resource_detectors/opentelemetry-resource-detectors.gemspec

```diff
@@ -9,7 +9,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'opentelemetry/resource/detectors/version'
 
 Gem::Specification.new do |spec|
-  spec.name        = 'opentelemetry-resource-detectors'
+  spec.name        = 'opentelemetry-resource_detectors'
```

Comment:
Does the name of this file need to change too?

Reply:
It does! Updated.